facilitator/src/intake.rs

```diff
@@ -284,6 +284,8 @@ mod tests {
     use assert_matches::assert_matches;
     use prio::{encrypt::PublicKey, server::ServerError, util::SerializeError};
 
+    const TRACE_ID: Uuid = Uuid::from_bytes([97; 16]);
```

Comment:
nit: this could be defined in `test_utils.rs` instead of being duplicated across test modules.